.travis.yml

```diff
@@ -30,3 +31,6 @@ script:
   - go vet ./management/...
   - test -z "$(golint ./Gododir/... | tee /dev/stderr)"
   - go vet ./Gododir/...
+  - set -e; for v in ./arm/... ./datalake-store/... ./Gododir/... ./management/... ./storage/...; do errcheck -ignoretests "$v"; done
```

Comment:
That's fair. I think I tried integrating errcheck long time ago manually and it just came with tons of changes which were not necessary (see my other comment). I just started to recall that...